### PR TITLE
Add www folder

### PR DESCRIPTION
**Pull request by PhantomCloak**

### CODE REVIEW

File: efurni-api/Services/ProductGroupingService/ProductGroupingService.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Caching.Memory;
 using efurni.Shared.Models;
+// ReSharper disable All
```
Comment (MFatihMAR) [inline efurni-api/Services/ProductGroupingService/ProductGroupingService.cs:8]: :(

File: efurni-api/Filter/AccesFilters.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Filters;
+// ReSharper disable All
```
Comment (MFatihMAR) [inline efurni-api/Filter/AccesFilters.cs:6]: :(

**Comment by PhantomCloak** [comment]
Removed unnecessary items from PR

File: efurni-api/Filter/AccesFilters.cs (hunk 2 above)
Comment (PhantomCloak) [reply efurni-api/Filter/AccesFilters.cs]: Fixed in a further commit

File: efurni-api/Services/ProductGroupingService/ProductGroupingService.cs (hunk 1 above)
Comment (PhantomCloak) [reply efurni-api/Services/ProductGroupingService/ProductGroupingService.cs]: Fixed in a further commit

**Pull request merged**